samples/azure-function/Dockerfile

```
@@ -10,10 +10,10 @@ COPY ["scripts/", "data/scripts/"]
 COPY ["samples/", "data/samples/"]
 
 WORKDIR /data
-RUN scripts/package/create_packages.sh src/CarbonAwareSDK.sln /packages && \
-    dotnet restore "samples/azure-function/function.csproj" && \
-    scripts/package/add_packages.sh samples/azure-function/function.csproj /packages && \
-    dotnet build "samples/azure-function/function.csproj" -c Release -o /app/build
+RUN pwsh -c scripts/package/create_packages.ps1 src/CarbonAwareSDK.sln /packages
```

```suggestion
RUN pwsh -c scripts/package/create_packages.ps1 src/GSF.CarbonAware/src/GSF.CarbonAware.csproj /packages && \
     dotnet restore "samples/azure-function/function.csproj" && \
     pwsh -c scripts/package/add_packages.ps1 samples/azure-function/function.csproj /packages && \
     dotnet build "samples/azure-function/function.csproj" -c Release -o /app/build
```

This will keep the image layers smaller (less `RUN`s). Also changing the invocation to use the GSF project instead of the solution.

---

Done!